x-pack/plugin/ql/src/test/java/org/elasticsearch/xpack/ql/expression/predicate/RangeTests.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.ql.expression.predicate;
+
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xpack.ql.expression.Literal;
+import org.elasticsearch.xpack.ql.tree.Source;
+import org.elasticsearch.xpack.ql.type.DataType;
+import org.elasticsearch.xpack.ql.type.DataTypes;
+import org.elasticsearch.xpack.ql.type.DateUtils;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+
+public class RangeTests extends ESTestCase {
+
+    private static final DataType K = DataTypes.KEYWORD;
```

Comment:
Not a big fan of this approach. `import static DataTypes.KEYWORD` and all other data types and use `KEYWORD`, `TEXT` and `DATETIME` everywhere.